hedera-node/hedera-network-admin-service-impl/src/test/java/com/hedera/node/app/service/networkadmin/impl/test/handlers/NetworkAdminHandlerTestBase.java

```diff
@@ -311,7 +314,7 @@ protected void givenValidFungibleToken(
                 deleted,
                 TokenType.FUNGIBLE_COMMON,
                 TokenSupplyType.INFINITE,
-                autoRenewAccountNumber,
+                AccountID.newBuilder().accountNum(autoRenewAccountNumber).build(),
```

Comment:
nit : I would suggest change `autoRenewAccountNumber` definition to `autoRenewId`

Reply:
Updated, thanks.